stf.test/test/saros/stf/test/categories/FailingTests.java

```diff
@@ -0,0 +1,9 @@
+package saros.stf.test.categories;
+
+/**
+ * Marks tests are always failing and have to be fixed! Create an issue "Fix STF failing test
+ * <Testname>" before using this marker
+ */
+public interface FailingTests {
```

Comment:
Should we really introduce such a category ? I can understand the other test category because it is sometimes hard to correctly simulate some test stuff but this category clearly represents a regression and so should be either fixed, or the test case has to be updated, or the commit that made the test case fail should be reverted.

Reply:
I introduce this category only for now because I want to run the stf tests on the master branch (with useful results that are not always red). I don't plan to use both categories in the future.

The plan is:
* Open an issue for each marked tests
* Fix failing tests and remove `FailingTests` category in the near future.
* And if possible do the same with FlakyTests.

If you like I can mark the category as deprecated. I just introduced it to distinguish the failing and the flaky tests for prioritization.